spec/models/manageiq/providers/amazon/cloud_manager/event_parser_spec.rb

```diff
@@ -1,4 +1,12 @@
 describe ManageIQ::Providers::Amazon::CloudManager::EventParser do
+  before :each do
+    _guid, _server, zone = EvmSpecHelper.create_guid_miq_server_zone
+    @ems                 = FactoryGirl.create(:ems_amazon, :zone => zone)
+
+    allow_any_instance_of(EmsEvent).to receive(:handle_event)
```

Comment:
`allow_any_instance_of` is the root of all evil. Chris and JoeR try hard to get eliminate it. Can do without it? Maybe inject an instance of EmsEvent into somewhere?